### ISSUE TITLE
ENGINX: Investigate MOXA

### ISSUE THREAD

**Issue by JamesKingWork** [ISISComputingGroup/IBEX#5592]
Both the Linmot's that control the jaws on ENGINX and the ENGINX positioner galils are not talking. These both talk on the MOXA 130.246.53.32 which we cannot get access to via the web portal, nor is it pingable, nor can moxautils contact it.

This ticket is to investigate the moxa when someone is on site and to fix the issue. The jaws and positioner are very important for ENGINX running.

## Acceptance criteria:

- ENGINX positioner is talking
- Linmot jaws are talking

**Comment by KathrynBaker** [comment]
Superseded by #5606 as there are a number of Moxa NPorts in need of looking at before cycle
